config/secrets.yml

```diff
@@ -246,9 +269,20 @@ production:
     security_token: <%= ENV['SALESFORCE_SECURITY_TOKEN'] %>
     consumer_key: <%= ENV['SALESFORCE_CONSUMER_KEY'] %>
     consumer_secret: <%= ENV['SALESFORCE_CONSUMER_SECRET'] %>
+    push_lead_enabled: <%= ENV['SALESFORCE_PUSH_LEAD_ENABLED'] || true %>
```

Comment:
It is impossible to set this value to false, since it has `|| true`.

Also, this applies to all the boolean ENV variables: ENV variables are always null or string so it may be a good idea to cast them to boolean using the activerecord boolean type caster thing. Alternatively, you can do this: `ENV['SALESFORCE_PUSH_LEAD_ENABLED'] != 'false'` if you want the default value to be true or `ENV['SALESFORCE_PUSH_LEAD_ENABLED'] == 'true'` if you want the default value to be false.

Reply:
Good catch Dante.  At one point I thought we had some utilities for helping with env vars.  Just found [Nenv](https://github.com/e2/nenv) which seems kinda cool too.

Reply:
changed - I should have noticed all your casting on other secrets 🤦🏼‍♂️

Reply:
Secrets aren't nil or a string, but environment variables are read as strings.